### ISSUE TITLE
Jet-Appointment 2ways google Calendar sync feature  

### ISSUE THREAD

**Issue by visheshjhadi** [CrocoBlock/suggestions#3517]
- if the feature is missing from the products.
----Yes 2 Ways google Calendar Sync for easy booking function. 

**Please, describe the feature request, making it clear to understand.**

I have to build a website for my client they want it as an appointment-based service for mental health. So I have created 2 post types one as provides and the other as except...and also set up the jetappointment. 
Now in the jet-engine form we a static form > in there we have a calendar option to book a slot. Now I want to 2ways sync google calendar with the booking slot. So that if there is an empty slot in the Google calendar that has to reflect in the jetappointment calendar. If now I manually add a meeting in google calendar ex: today 5 pm is a book. ( I did it manually).
I want the appointment calendar to show 5 pm as blocked or it should not be available for booking.

With This feature, appointment booking will be much more automated and easy to handle. 

**Add an example of where this feature is already implemented. **
https://wpamelia.com/ this might be your competitor, this plugin provides 2-way google calendar sync. have a look at it.

Provide a detailed explanation of where this feature should be used, and if there are any alternatives already. 

This feature can be added to both jetappointment and also jet booking.

Feel free to add any videos, links, screenshots to illustrate the requested feature.

**https://wpamelia.com/**

Please, note, that the feature requests can be accepted and processed only if they’re written in English.

**Comment by jtrzeff** [comment]
Yes, I am using WPBase https://wp-base.com/ and the 2-way sync with Google Calendar is essential for our business coaches. We would much rather use JetAppointment since we've invested into the JetEngine environment 

<img width="1339" alt="image" src="https://user-images.githubusercontent.com/87763995/126491665-d63e03cf-df5a-4d29-8caa-59dbb2bb011f.png">

**Comment by catherinewiltshire** [comment]
This is such a needed feature.
I just thought I'd pull in a few other threads where 2 way sync for calendar has been requested previously.

#3503  
#3012  
#2543 

I know there are more threads on this but I can't find them at the moment. It would be great to hear from crocoblock if they are planning on adding this (and when!) or if we should invest in a different plugin.

**Comment by tofulabjp** [comment]
This is seriously needed. 
I can't recommend people to use this plugin without this feature.
We can probably accomplish this by using Zapier/Integromat but it's not intuitive.

**Comment by IFTTD** [comment]
Definitely needed. Alternative would be to allow a away to insert an Appointment using an Endpoint or WP API. Currently it seems it is only a one way sync with google calendar. This causes issues when a manual entry into the google calendar is made and we get double bookings.

**Comment by ouun** [comment]
On the Crocoblock website I read "synchronize your appointment schedule". That was the reasony why I wanted to get the plugin. Hope that this gets added soon as I agree that the plugin is a bit useless without it.

**Comment by davidwatkins2** [comment]
This is needed!

**Comment by Zszywaczyk** [comment]
+1

**Comment by davidwatkins2** [comment]
Definitely needed! Please add this! Without this clients prefer to use other plugins with the two way syncing because it makes it easier to accept or deny appointments from their calendar/ email. 

WP Amelia is a great plugin to reference. If Jet Engine add this feature it would blow competition out of the water.

**Comment by klwdp** [comment]
This is needed badly. Is there any plan to add this as it is a deal breaker for my clients?

**Comment by findatutor** [comment]
+1, Could we al least know if this is being considered? Cause if not I might have to switch to another system of plugins.

**Comment by trofimartya** [comment]
+1
Also Outlook 2-way sync would be awesome idea.

**Comment by zehjotkah** [comment]
this is so important.

**Comment by Topicimes** [comment]
+1

**Comment by edelm4nn** [comment]
+1

**Comment by Microsmith-1** [comment]
Any mention of this from the CB team and any idea if they are considering it? Ive asked many times and got no reply. ive had to walk away from JetAppointments, We cant keep hanging around waiting for things to happen, im now using Appointedd which has a ton of integration, but id still rather use JetEngine cos ive invested in it.

**Comment by Creatify5** [comment]
This was one of the reasons i bought Crocoblock, there was an appointment and booking plugin, so i don't have to buy another one. Without this 2 way sync i still need to buy another plugin.....
How is this after 3 years still not integrated but JetBooking has it already..

**Comment by johnkraczek** [comment]
+1 here

**Comment by MjHead** [comment]
Added with 2.2.0 update
